src/pages/Landing/index.tsx

```diff
@@ -384,15 +384,27 @@ export default function Landing() {
               </LearnMoreContainer>
             </ContentContainer>
             <AboutContentContainer isDarkMode={isDarkMode}>
-              <CardGrid cols={2} ref={cardsRef}>
-                {MAIN_CARDS.map(({ darkBackgroundImgSrc, lightBackgroundImgSrc, ...card }) => (
-                  <Card
-                    {...card}
-                    backgroundImgSrc={isDarkMode ? darkBackgroundImgSrc : lightBackgroundImgSrc}
-                    key={card.title}
-                  />
-                ))}
-              </CardGrid>
+              {shouldDisableNFTRoutes ? (
```

Comment:
To avoid duplication, how would you feel about doing this logic in an earlier useMemo, something like

```
const cards = useMemo(() => MAIN_CARDS.filter((card) => shouldDisableNFTRoutes ? !card.to.startsWith('/nft') : true), [shouldDisableNFTRoutes])
```

And then you can set the `cols={cards.length}`.

Reply:
That makes sense! Just updated